libs/libpugiutil/src/pugixml_loc.cpp

```diff
@@ -30,7 +30,7 @@ void loc_data::build_loc_data() {
 
     std::ptrdiff_t offset = 0;
 
```

Comment:
Thanks.  I think we should make a defined constant with a comment for this magic number though, e.g. 
#define VTR_LARGE_BUF 1048576  // Read large files like the arch.xml with large buffers for speed.

Reply:
Should have a comment on why too.